public/v4/apps/bookstack.yml

```diff
@@ -0,0 +1,60 @@
+captainVersion: 4
+services:
+    $$cap_appname-mariadb:
+        image: linuxserver/mariadb:110.4.14mariabionic-ls77
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_DB_PASS
+            MYSQL_DATABASE: bookstack
+            MYSQL_USER: bookstack
+            MYSQL_PASSWORD: $$cap_DB_PASS
+            PGID: 1000
+            PUID: 1000
+            TZ: $$cap_timezone
+        volumes:
+            - $$cap_appname-mariadb-config:/config
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        environment:
+            DB_HOST: srv-captain--$$cap_appname-mariadb
+            DB_USER: bookstack
+            DB_PASS: $$cap_DB_PASS
+            DB_DATABASE: bookstack
+            DB_PORT: 3306
+            PGID: 1000
+            PUID: 1000
+            TZ: $$cap_timezone
+            APP_URL: $$cap_appurl
+        image: linuxserver/bookstack:v0.29.3-ls99
+        restart: always
+        volumes:
+            - $$cap_appname-config:/config
+        caproverExtra:
+            containerHttpPort: '80'
```

Comment:
80 is the default port, no need to define it explicitly

Reply:
Removed in 1d3a7f4